signac/core/utility.py

```diff
@@ -78,6 +88,12 @@ def __repr__(self):
         return "Version({})".format(",".join((f"{k}={v}" for k, v in self.items())))
 
 
+@deprecated(
+    deprecated_in="1.8",
+    removed_in="2.0",
+    current_version=__version__,
+    details="Use packaging.version.Version instead.",
```

Comment:
```suggestion
    details="Use packaging.version.parse instead.",
```

Reply:
Neither `version.Version` nor `version.parse` really has the same signature as this object. I suppose `version.parse` is the more useful alternative to indicate to users, but `version.Version` is probably the more accurate analog to this object.